terraform/cloudinit/kali-instance.yaml

```
@@ -13,6 +13,7 @@ users:
     ssh_authorized_keys:
       - ${ssh-pub-key-1}
       - ${ssh-pub-key-2}
+      - ${ssh-pub-key-3}
```

https://www.terraform.io/docs/configuration/expressions.html#directives
Loops in templates
```suggestion
    ssh_authorized_keys:
    %{ for key in ssh-keys ~}
      - ${key}
    %{ endfor ~}
```